cscs-checks/compile/libsci_resolve.py

```diff
@@ -8,48 +8,47 @@
 
 
 class LibSciResolveBaseTest(rfm.CompileOnlyRegressionTest):
-    def __init__(self):
-        self.sourcesdir = 'src/libsci_resolve'
-        self.sourcepath = 'libsci_resolve.f90'
-        self.valid_systems = ['daint:login', 'daint:gpu',
-                              'dom:login', 'dom:gpu']
-        self.modules = ['craype-haswell']
-        self.maintainers = ['AJ', 'LM']
-        self.tags = {'production', 'craype'}
+    sourcesdir = 'src/libsci_resolve'
+    sourcepath = 'libsci_resolve.f90'
+    valid_systems = ['daint:login', 'daint:gpu', 'dom:login', 'dom:gpu']
+    modules = ['craype-haswell']
+    maintainers = ['AJ', 'LM']
+    tags = {'production', 'craype'}
+
+    @run_after('setup')
+    def set_postbuild_cmds(self):
+        self.postbuild_cmds = [f'readelf -d {self.executable}']
 
 
-@rfm.parameterized_test(['craype-accel-nvidia35'], ['craype-accel-nvidia60'])
+@rfm.simple_test
 class NvidiaResolveTest(LibSciResolveBaseTest):
-    def __init__(self, module_name):
-        super().__init__()
-        self.descr = f'Module {module_name} resolves libsci_acc'
-        self.build_system = 'SingleSource'
+    accel_nvidia_version = parameter(['35', '60'])
+    valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu']
```

Comment:
cray-libsci_acc does not support PrgEnv-nvidia,
but it can be tested with:

```
module unload PrgEnv-cray
module load PrgEnv-nvidia
module load cdt/21.05
module load craype-haswell
ftn libsci_resolve.f90 -o ./NvidiaResolveTest_60 -L$NVIDIA_PATH/math_libs/lib64 -lnvblas
```

Maybe worth having a new test ?

Reply:
Yes, that could be either another test in the same file or new test.